main.py

```diff
@@ -186,67 +186,91 @@ async def set_playing(status):
     await client.change_presence(activity=discord.Game(name=status))
 
 
-async def give_random(channel, number_of_issues):
-    """
-    @TODO: improve algorithm to save queries
-    1. get first batch of github_max (100), find total number
-    2. if total < requested, return all
-    3. Generate requested random numbers
-    4. get issues till max(generated_list)
-    5. return them
-    """
-    await set_playing('Finding Issues')
-    error_channel = error_handler(channel)
-    async with channel.typing():
-        issues = [x async for x in get_issues(config["repo"],
-                                              exclude_labels=['status: stale'],
-                                              error_channel=error_channel)]
-        reply = discord.Embed(color=discord.Color.purple())
-        reply.title = f"{number_of_issues} random picks out of {len(issues)}:"
-
-        chosen_issues = random.choices(issues, k=number_of_issues)
-        reply.description = compose_message(beautify_issues(chosen_issues))
-        if len(chosen_issues) < number_of_issues:
-            reply.set_footer(text="There wasn't enough...")
-    await channel.send(embed=reply)
-    await set_playing('The Waiting Game')
-
-
-async def review_q(channel, number_of_issues, author=None):
-    await set_playing('On The Cue')
-    error_channel = error_handler(channel)
-    async with channel.typing():
-        issues = get_issues(config["repo"], pull_request=True,
-                            exclude_labels=['status: stale'],
-                            include_labels=['needs: code review'],
-                            sort='updated', ascending_order=True,
-                            error_channel=error_channel)
-        reply = discord.Embed(color=discord.Color.purple())
-
+class SendReply:
+    command_type=None
+    channel=None
+    number_of_issues=None
+    author=None
+    
+    git_config=None
+    discord_config=None
+
+    issues=[]
+    reply=None
+    chosen_issues=[]
+
+    def __init__(self, command_type, channel, number_of_issues, author):
+        self.command_type=f"{command_type}"
+        self.channel=channel
+        self.number_of_issues=number_of_issues
+        self.author=author
+
+        command_config=read_config(command_config_filepath)
+        self.git_config=command_config["type"][command_type]["git"]
+        self.discord_config=command_config["type"][command_type]["discord"]
+
+    
+    def issues_generator(self):
+        self.issues=[x async for x in get_issues(self.git_config["repository"], self.git_config["closed"],
```

Comment:
* Have sensible defaults to reduce data in config file
* Use `**` to expand a dictionary for kwargs

Reply:
Adds a lot of arguments:
```
def issues_generator(repository, closed, pull_request, include_labels, 
                     exclude_labels, sort, ascending_order):
    self.issues=[x async for x in get_issues(repository, closed, pull_request, 
                                             include_labels, exclude_labels,
                                             sort, ascending_order)]
```
```
issues_generator(**self.git_config)
```

Reply:
No. It'll be something similar to
```python
def issue_generator(self):
    self.issues=[x async for x in get_issues(**self.config)]
```

Reply:
My bad. 
Although we should default to Python>3.6 since it guarantees order preservation in kwargs. https://docs.python.org/3/whatsnew/3.6.html#whatsnew36-pep468

Reply:
Not needed if repository is also made a keyword argument with a default value of "github/training-kit" since then all arguments will be keyword arguments

Reply:
Okay. I'll add the default value in `get_issues`